lib/verify_docs.rb

```diff
@@ -0,0 +1,7 @@
+class VerifyDocs
+  def self.call
+    readme_content = File.read("README.md").lines[2..-1]
+    docs_content = File.read("docs/index.md").lines[4..-1]
```

Comment:
The absolute line indices make this somewhat fragile but I think it's sufficient for now.

Reply:
One viable option for this would be to add a comment to both files indicating where the shared content begins (which also serves as a reminder for people editing it) and ignore anything before that comment.